icicle-compiler/test/Icicle/Test/Arbitrary/NanEq.hs

```diff
@@ -117,7 +117,9 @@ instance NanEq Time where
 
 instance NanEq OutputName
 
-instance NanEq Namespace
+instance NanEq Namespace where
+  nanEq =
+    (==)
```

Comment:
I'm just curious, did this need to be changed because the constructor is no longer exported?